### PR TITLE
Child Home Visit Essential newborn care breastfeeding Module

### LINKED ISSUE #15: Essential New Born Care: Breastfeeding
[Essential Newborn Care:Breastfeeding](https://docs.google.com/spreadsheets/d/1dr8uhaF9KN09DzcQf6-l_VsC6jXCQ5ddj1cKnuymkto/edit#gid=179989320&range=51:51)

### PR DESCRIPTION
**Pull request by devcolonel**
Close #15

### CODE REVIEW

**Review by issyzac — DISMISSED** [review]
LGTM

File: opensrp-chw/src/nacp/assets/recurring_service_types.json
Comment (gosso22) [inline opensrp-chw/src/nacp/assets/recurring_service_types.json:1]: @devcolonel - Please check the frequency on the breastfeeding content as per data dictionary and update here. https://docs.google.com/spreadsheets/d/1dr8uhaF9KN09DzcQf6-l_VsC6jXCQ5ddj1cKnuymkto/edit#gid=179989320&range=51:51 - look on specifically on cell 56. I think the definitions you have on this file here are monthly.

**Review by gosso22 — CHANGES_REQUESTED** [review]
Please revise on the recurring service definitions.

**Review by gosso22 — DISMISSED** [review]
LGTM!

**Review by gosso22 — DISMISSED** [review]

File: opensrp-chw/src/nacp/assets/rule/child_hv_breastfeeding_relevance.yml [hunk 1]
```diff
@@ -0,0 +1,91 @@
+---
+name: step1_breastfeed_current_yes
+description: Check if Child Growth Present to do Growth Monitoring
```
Comment (gosso22) [inline opensrp-chw/src/nacp/assets/rule/child_hv_breastfeeding_relevance.yml:3]: @devcolonel - the description here is misleading can you change to reflect this rule?

**Review by gosso22 — APPROVED** [review]

**Pull request merged**